### PR TITLE
added stack

### PR DESCRIPTION
**Pull request by Gumanpratapsingh**
added new Data Structure in python

### CODE REVIEW

**Comment by karthik2265** [comment]
make it a class and add push and pop methods.

**Comment by Gumanpratapsingh** [comment]
Please review and add hacktoberfest label also so that it can be tracked

File: Data-Structures/stack/stack.py [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+        self.items = []
+ 
+    def is_empty(self):
+        return self.items == []
+ 
+    def push(self, data):
+        self.items.append(data)
+ 
+    def pop(self):
+        return self.items.pop()
```
Comment (karthik2265) [inline Data-Structures/stack/stack.py:12]: check items is not empty before popping

File: Data-Structures/stack/stack.py [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
+ 
+ 
+s = Stack()
+while True:
```
Comment (karthik2265) [inline Data-Structures/stack/stack.py:16]: dont need this taking input from user and stuff. remove this while loop

**Review by karthik2265 — CHANGES_REQUESTED** [review]

File: Data-Structures/stack/stack.py (hunk 2 above)
Comment (Gumanpratapsingh) [reply Data-Structures/stack/stack.py:16]: should i go with the static one ?
like this ->
    stack.push(1)
    stack.push(2)
    stack.push(3) 
and also popping with the static only
means hardcoded ?

**Comment by Gumanpratapsingh** [comment]
I have added the trees with its traversal 
Can you check Please 
I have remove the stack as I didn't get about that while loop.
Should we hardcoded that push and pop ? without getting the choice from user 
Please check the trees DS

File: Data-Structures/stack/stack.py (hunk 2 above)
Comment (karthik2265) [reply Data-Structures/stack/stack.py]: @Gumanpratapsingh  yes, hardcoded and try to explain with comments what's happening, thanks for contributing.

**Comment by Gumanpratapsingh** [comment]
Hello Sir,
Can you check?
I have added the requested changes .

**Comment by Gumanpratapsingh** [comment]
Ok Thanks
Please add the Hacktober-accepted label so it can be tracked.

**Pull request merged**